README.md

```diff
@@ -112,7 +112,7 @@ Fixes the given *node*’s position to ⟨*x*,*y*⟩ and the velocity to zero, a
 
 <a name="simulation_unfix" href="#simulation_unfix">#</a> <i>simulation</i>.<b>unfix</b>(<i>node</i>)
```

Comment:
Need square brackets around *node* to indicate that the argument is now optional.